cmd/kured/main.go

```diff
@@ -635,6 +635,12 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 				if err != nil {
 					log.Errorf("Unable to uncordon %s: %v, will continue to hold lock and retry uncordon", node.GetName(), err)
 					continue
+				} else {
+					if notifyURL != "" {
+						if err := shoutrrr.Send(notifyURL, fmt.Sprintf("Node: %s rebooted&uncordoned successfully!", nodeID)); err != nil {
```

Comment:
nit: add a space so it reads "rebooted & uncordoned successfully!"

Reply:
Done :+1: